src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/ListeningSettingsModal.razor

```diff
@@ -0,0 +1,88 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@implements IModalView<ListeningSettingsModal.Model>
+@using ActualChat.Kvas
+@using ActualChat.Audio
+@inherits ComputedStateComponent<ListeningSettingsModal.ComputedModel>
+@{
+    var m = State.Value;
+    var listeningMode = m.ListeningMode;
+    var defaultTimeout = (int)Math.Floor(AudioSettings.IdleListeningTimeout.TotalMinutes);
+}
+<DialogFrame Title="Listening settings" Class="transcription-options-modal modal-sm" HasCloseButton="false">
+    <Body>
+    <FormBlock>
+        <Tile>
+        <TileItem Click="() => UpdateListeningMode(ListeningMode.Default)">
```

Comment:
Candidate for template func

Reply:
I don't get you, please provide some context or an example